linetools/lists/linelist.py

```diff
@@ -329,14 +329,22 @@ def set_extra_columns_to_datatable(self, abundance_type='solar', ion_correction=
                                        redo=False):
         """Sets new convenient columns to the self._data QTable. These will be useful
         for sorting the underlying data table in convenient ways, e.g. by expected
-        relative strength.
-        These new column include:
+        relative strength, abundance, etc.
+
+        * For atomic transitions these new column include:
             - `ion_name` : HI, CIII, CIV, etc
             - `log(w*f)` : np.log10(wrest * fosc)  # in np.log10(AA)
             - `abundance` : either [`none`, `solar`]
             - `ion_correction` : [`none`]
             - `rel_strength` : log(w*f) + abundance + ion_correction
 
+        * For molecules a different approach is done:
+            - `ion_name` : B0-0P, C6-0, etc.
+            - `rel_strength`: We have only three arbitrary levels: [1, 50, 100]
+                    100 is for Jj={0,1} and Jk={0,1}
```

Comment:
Reviewing H2 a bit, I see that the key is really Jk.  I'd do this:

100 for Jk = 0,1
50 for Jk = 2,3
1 otherwise